x/operator/keeper/usd_value.go

```diff
@@ -1,9 +1,13 @@
 package keeper
 
 import (
+	"errors"
 	"fmt"
 
 	assetstype "github.com/ExocoreNetwork/exocore/x/assets/types"
+	delegationkeeper "github.com/ExocoreNetwork/exocore/x/delegation/keeper"
+	delegationtype "github.com/ExocoreNetwork/exocore/x/delegation/types"
+	oracletypes "github.com/ExocoreNetwork/exocore/x/oracle/types"
```

Comment:
oracletype
keep the uniform style is better.

Reply:
Thanks

Reply:
This will be fixed in the next PR.